xen/arch/x86/mm/p2m.c

```diff
@@ -1232,11 +1228,7 @@ int clear_identity_p2m_entry(struct domain *d, unsigned long gfn_l)
     {
         if ( !need_iommu(d) )
             return 0;
-<<<<<<< HEAD
-        return iommu_unmap_page(d, gfn_l);
-=======
-        return iommu_unmap_pages(d, gfn, 0);
->>>>>>> d530dd9... iommu: Add extra order argument to the IOMMU APIs and platform callbacks
+        return iommu_unmap_page(d, gfn_l, 0);
```

Comment:
This is not correct as well. You should have left iommu_unmap_pages.
I am wondering have you ever tried to build this stuff under x86?

Reply:
No, I didn't.